projects/handofgod/index.js

```diff
@@ -1,27 +1,82 @@
 const { sumTokensExport } = require("../helper/unwrapLPs");
-
+const sdk = require("@defillama/sdk");
 
 const hogGenesisAddress = '0x2e585b96a2ef1661508110e41c005be86b63fc34'
-const tokens = [
-  "0xb1e25689D55734FD3ffFc939c4C3Eb52DFf8A794",  // OS 19% 
-  "0x79bbF4508B1391af3A0F4B30bb5FC4aa9ab0E07C",  // Anon 10% 
-  "0x44E23B1F3f4511b3a7e81077Fd9F2858dF1B7579",  // Mclb 9% 
-  "0xA04BC7140c26fc9BB1F36B1A604C7A5a88fb0E70",  // SWPx 11% 
-  "0xE5DA20F15420aD15DE0fa650600aFc998bbE3955",  // stS 7% 
-  "0xd3DCe716f3eF535C5Ff8d041c1A41C3bd89b97aE",  // scUSD 7% 
-  "0x4EEC869d847A6d13b0F6D1733C5DEC0d1E741B4f",  // Indi 4% 
-  "0x9fDbC3f8Abc05Fa8f3Ad3C17D2F806c1230c4564",  // Goglz 4% 
-  "0x2D0E0814E62D80056181F5cd932274405966e4f0",  // Beets 2%
+const rewardPoolAddress = '0xa7141905e2972c295577882552bede5406daf5ec'
+const masonryAddress = "0xf156C473F4423B7013C612Da0ACD9DDcC33DB3EA"
+const ghogToken = "0x0e899dA2aD0817ed850ce68f7f489688E4D42D9D"
+const ghogPool = "0xd1cb1622a50506f0fddf329cb857a0935c7fbbf9"  // GHOG/OS pool
+const osToken = "0xb1e25689d55734fd3fffc939c4c3eb52dff8a794"   // OS token
+
+const genesisTokens = [
+  "0xb1e25689d55734fd3fffc939c4c3eb52dff8a794",  // OS 19% 
+  "0x79bbf4508b1391af3a0f4b30bb5fc4aa9ab0e07c",  // Anon 10% 
+  "0x44e23b1f3f4511b3a7e81077fd9f2858df1b7579",  // Mclb 9% 
+  "0xa04bc7140c26fc9bb1f36b1a604c7a5a88fb0e70",  // SWPx 11% 
+  "0xe5da20f15420ad15de0fa650600afc998bbe3955",  // stS 7% 
+  "0xd3dce716f3ef535c5ff8d041c1a41c3bd89b97ae",  // scUSD 7% 
+  "0x4eec869d847a6d13b0f6d1733c5dec0d1e741b4f",  // Indi 4% 
+  "0x9fdbc3f8abc05fa8f3ad3c17d2f806c1230c4564",  // Goglz 4% 
+  "0x2d0e0814e62d80056181f5cd932274405966e4f0",  // Beets 2%
 ]
 
+async function masonryTVL(api) {
+  // Get GHOG balance in masonry
+  const masonryGhog = await api.call({  
+    target: ghogToken,
+    params: [masonryAddress],
+    abi: 'erc20:balanceOf'
+  })
+
+  // Get reserves from GHOG/OS pool to calculate price
+  const [token0, token1, reserves] = await Promise.all([
```

Comment:
We have added pricing support for GHOG so this logic is no longer required. GHOG balance should be exported under `staking` and not `tvl` as it is your own token